src/cmd-koji-upload

```diff
@@ -106,6 +106,7 @@ class Build(_Build):
         self.platform = "koji"
         # Use a tempdir in builds/ because we want to use large scratch space
         self._tmpdir = tempfile.mkdtemp(prefix="koji-build", dir="builds/")
+        self._state_file_tpl = self._tmpdir + "/cosa-cmd-koji-upload-{name}-{version}-{release}"
```

Comment:
Should this be `f"/cosa-cmd-koji-upload-{name}-{version}-{release}"`?

Reply:
Yes, this file will store the reservation data if asked to use a file, we can change this long name though.

Reply:
Ah, right, I missed that the rendering happens later. 

CI is failing, otherwise, this looks good.

Reply:
Ok, I found the issue in the CI. It will need to way the fix.
It is not getting the right fedora-coreos-config version to build, it is pointing to testing-devel instead of rhcos-4.7. I'm checking it